Login/login.css

```diff
@@ -0,0 +1,74 @@
+body{
+    margin: 50; 
+    background: rgb(2, 133, 255)
+}
+.main{
+    align-items: center;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+}
+.logo{
+    height: 50px;
+    width: 50px;
+    position: fixed;
+    margin: -156px 0px 0px 0px;
```

Comment:
also whenever use absolute use top, bottom, right and left proprieties don't use margin